src/screens/Home/header.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import logo from 'images/logo_white.png';
+import userImage from 'images/user-placeholder.jpg';
+
+const Header = () => {
+  return (
+    <div className="home-header">
+      <div className="home-header__left">
```

Comment:
@rafayet-monon I think we shouldn't use `__left` and `__right` for naming this.
In case we want to re-position this to `right` with some screen so this class name will not correct anymore 😅